cmd/indexer/internal/processor/sequential.go

```diff
@@ -17,6 +17,16 @@ type Sequential struct {
 	blockClient    clients.BlockByNumber
 }
 
+func New(protocol, network string, store datastore.SyncStore, proc actions.Block, client clients.BlockByNumber) *Sequential {
```

Comment:
The could be other types of processors in this package over time `NewSequential` would be a clearer name